### DISCUSSION TITLE
Would it be nice to release sdcard image?

### DISCUSSION THREAD

**Discussion by xbing6** [parkrrrr/bdio#15] [Q&A] [answered]
Would it be nice to release sdcard image?

A) If one has Ron's un-released HID Braille software, he / she can try on Windows 10.
B) I can try on my macOS Big Sur.
C) If BRLTTY supports HUTRR-78 someday, people can try on Ubuntu / Linux Host.

There is below info in README:
------
Some way to write to a Micro SD card from the computer on which you're building BDIOHID. This means software and hardware. Personally, my build machine is an Ubuntu VM running in Oracle VirtualBox on top of a Windows 10 host, so I use a cheap glittery purple USB-to-MicroSD adapter I picked up at Office Depot for hardware, and Balena Etcher for software. Use what you're comfortable with. (I would probably use Unix dd, but the Windows host autodetects the FAT boot partition and tries to mount the drive, interfering with dd.)

Put the MicroSD card in the NanoPI NEO, plug your Braille display into the NEO's USB A port, plug the USB Micro-B cable into the NEO's "OTG" port, and plug the other end of that cable into your computer.

**Comment by parkrrrr** [comment]
That's a good idea. It's definitely something I had always planned to do. It's slightly less useful at the moment because the database only has entries for the one Braille display I've been testing with (and maybe one or two that don't work, in the sense that the hardware I have for those two displays doesn't seem to work) but I can definitely upload it. It may not happen before the holiday, though.

**Comment by parkrrrr** [comment]
An update on this: I have recently been in the process of bringing a new, more powerful machine online, and so as part of that process I have decided to move the development of bdio to that machine. That means I got to debug my own build instructions (which have A LOT of gaps I intend to fix.) 

So, once I get the build errors due to missing libraries like sqlite fixed, I'll get some binaries up. I've done about half of the work on generalizing the support for legacy devices, too, so hopefully soon after that I'll have a new build that supports more than just the two or three devices that are currently supported.

**Comment by parkrrrr** [comment, marked as answer]
This took longer than expected, both because I have been short on time and because my build instructions in the README are seriously lacking, and I had to figure out what I was missing. But hey, we got a new issue for the bug tracker, so that's something, right?

Anyway, there's a new file [in this folder](https://github.com/parkrrrr/bdio/tree/master/images) that is a freshly-built image for the NanoPi Neo that should support the HIMS SyncBraille, HumanWare Brailliant BI 40, and Baum VarioUltra. 

Let me know if you have trouble with it and I'll do what I can to help.

**Reply by parkrrrr** [reply]
Notes on this image:

- There is no root password set. If you hook a serial interface up to the UART, you'll have a terminal with root access.
- ~~This still has the testing USB VID/PID. I have an official PID now, but I haven't updated the database in the bdio source.~~
- ~~I tested this on a NanoPi Neo LTS. For some reason it's not working on my Neo Core plugged in to the carrier board. I'll update the image when I figure out why.~~ _I think this is a problem with that board, not with the build._
- Bonus: I added `lsusb` to this image, because I needed it to debug that previous bullet point. But maybe it'll be useful.
